doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -211,6 +211,17 @@ Memory or Huge Pages are not considered by the Topology Manager. This can be don
 
 [Device Plugin Framework](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/) advertises device hardware resources to kubelet with which vendors can implement plugins for devices that may require vendor-specific activation and life cycle management, and securely maps these devices to containers.
 
+Figure 3-2 shows in four steps how device plugins operate on a Kubernetes node:
+* 1: During setup, the cluster administrator as human or operator in software (more in [3.2.1.8 Operator Pattern](chapter03.md#3218-operator-pattern)) knows or discovers (as per [3.2.1.5 Node Feature Discovery](chapter03.md#3215-node-feature-discovery)) what kind of devices are present on the different machines, selects which devices to enable and deploys the associated device plugins.
```

Comment:
@petorre @arkadykanevsky 
I don't think "human or software" is necessary
In other bullets, "node" is used
```suggestion
* 1: During setup, the cluster administrator (more in [3.2.1.8 Operator Pattern](chapter03.md#3218-operator-pattern)) knows or discovers (as per [3.2.1.5 Node Feature Discovery](chapter03.md#3215-node-feature-discovery)) what kind of devices are present on the different nodes, selects which devices to enable and deploys the associated device plugins.
```